### ISSUE TITLE
Accesibility errors WCAG 2 using https://wave.webaim.org/ validator

### ISSUE THREAD

**Issue by danos5d5** [woocommerce/woocommerce#28837]
## Describe the bug
Clearly describe the issue. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

Isolating the problem (mark completed items with an [x]):
- [x ] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x ] I can reproduce this bug consistently using the steps below.

## To Reproduce
1. Go to https://themes.woocommerce.com/storefront/product-category/clothing/bags/
2. Copy the address
3. Go to https://wave.webaim.org/
4. Paste the adress from step 1 in the form field and run the validation
5. You get two "Missing form label" errors 

## Screenshots
![image](https://user-images.githubusercontent.com/6008212/104819466-658ab700-5836-11eb-81e9-e4b366ddb92f.png)

## Expected behavior
The validation should be error free and in compliance with [WCAG2](https://www.w3.org/TR/WCAG20/)

### Browser Environment
- Platform: Windows 
- Browser(s): Chrome

### WordPress Environment

```
The test was made on the official theme's page
```
</details>

**Comment by nerrad** [comment]
It looks like this was fixed at some point since the time this was reported.

![Image](https://github.com/woocommerce/woocommerce/assets/1429108/90647457-9f34-48f7-9a3c-9d5c4c218b4c)
